tests_integration/snowflake_connector.py

```diff
@@ -10,19 +11,41 @@
 
 
 @pytest.fixture(scope="session")
-def snowflake_session():
+def create_database():
+    database_name = f"db_{uuid.uuid4().hex}"
+
+    config = {
+        "application": "INTEGRATION_TEST",
+        "account": _get_from_env("ACCOUNT", True),
+        "user": _get_from_env("USER", True),
+        "password": _get_from_env("PASSWORD", True),
+        "host": _get_from_env("HOST", True),
+        "role": _get_from_env("ROLE", True),
+    }
+    config = {k: v for k, v in config.items() if v is not None}
+    connection = connector.connect(**config)
+
+    connection.execute_string(f"create database {database_name}")
+    os.environ[f"{_ENV_PARAMETER_PREFIX}DATABASE"] = database_name
+
+    yield database_name
+
+    connection.execute_string(f"drop database {database_name}")
+    del os.environ[f"{_ENV_PARAMETER_PREFIX}DATABASE"]
```

Comment:
```suggestion
    del os.environ[f"{_ENV_PARAMETER_PREFIX}_DATABASE"]
```
Imho this is more readable than using `_` suffix in the prefix

Reply:
Changed